### PR TITLE
Info card 

### PR DESCRIPTION
**Pull request by cyra997**
First version of the info card with header and text has been made.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -9,11 +9,20 @@ const App: React.FunctionComponent = () => {
...
       align="center"
-      backgroundColor="blue.700"
+      backgroundColor="blue.900"
     >
-      <Heading fontSize="4xl" color="white">
-        Let's get building...
+
+      <Flex
+      flexDir="column"
+      w="250"
```
Comment (jimpala) [inline src/App.tsx]: Use design system token for the width here (e.g: "sm", "lg", "3xl")

**Review by jimpala — CHANGES_REQUESTED** [review]
Please implement changes to sizing given here!

File: src/App.tsx (hunk 1 above)
Comment (jimpala) [reply src/App.tsx]: Smaller widths (like padding) use numbers in the Chakra design system theme file, larger widths use "sm", "3xl" etc

Comment (cyra997) [reply src/App.tsx]: The requested change has been made.

**Review by jimpala — APPROVED** [review]
Looks good to me!

**Pull request merged**